libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -304,6 +305,10 @@ void AP_Notify::add_backends(void)
 #endif
 #endif // Noise makers
 
+    // Always add AnyLED backend
```

Comment:
This needs to be one of the Notify_LED_Type's and live inside the switch above.

Reply:
there's no special LED types to direct control, just send the RGB value to I2C，I used one Arduino Pro Mini as I2C slaver to recive the RGB value，then Arduino to control any other RGB LEDs like WS2812b ...

the I2C slaver like this: https://github.com/ningjianggeek/AnyLED_External_I2C